dvc/dagascii.py

```diff
@@ -61,9 +63,10 @@ def __init__(self, cols, lines):
     def draw(self):
         """Draws ASCII canvas on the screen."""
         if sys.stdout.isatty():  # pragma: no cover
-            from asciimatics.screen import Screen
-
-            Screen.wrapper(self._do_draw)
+            lines = []
+            for line in self.canvas:
+                lines.append("".join(line))
+            pydoc.pager(os.linesep.join(lines))
```

Comment:
could be a one-liner

```python
pydoc.pager(os.linesep.join(map("".join, self.canvas)))
```

Reply:
multiliner is easier to print/debug if needed, but i can make it one liner, should I?
btw, thanks for this tip

Reply:
@xliiv, what about `lines = map("".join, self.canvas)` and then `pydoc.pager(os.linesep.join(lines))` ?